guides/common/modules/ref_utilities-for-processing-log-information.adoc

```diff
@@ -1,37 +1,52 @@
-[id="Utilities_for_Collecting_Log_Information_{context}"]
-= Utilities for Collecting Log Information
+[id="Utilities_for_Processing_Log_Information_{context}"]
+= Utilities for Processing Log Information
 
 You can collect information from log files to troubleshoot {Project}.
 
 ifdef::foreman-el,katello,satellite[]
 sosreport::
 The `sosreport` command collects configuration and diagnostic information from a Linux system, such as the running Kernel version, loaded modules, running services, and system and service configuration files.
 This output is stored in a tar file located at `/var/tmp/__sosreport-XXX-20171002230919.tar.xz__`.
++
 For more information, run `sosreport --help` or see https://access.redhat.com/solutions/3592[_What is a sosreport and how can I create one?_].
 endif::[]
-
 ifdef::foreman-deb,orcharhino[]
 foreman-debug::
 The `foreman-debug` command collects configuration and log file data for {ProjectName}, its back-end services, and system information.
-This information is collected and written to a tar file.
-By default, the output tar file is located at `/tmp/__foreman-debug-xxx.tar.xz__`.
-
+This output is stored in a tar file.
+By default, the tar file is located at `/tmp/__foreman-debug-xxx.tar.xz__`.
++
 Additionally, the `foreman-debug` command exports tasks run during the last 60 days.
```

Comment:
Should this also be changed to `orcharhino-debug`? Would an attribute help here?

Reply:
Sorry, I should have noticed there was more. I'll take a closer look at the whole file, an attribute might indeed be a good solution.

Reply:
Addressed in the latest commit: a new attribute named :project-debug: that resolves to either foreman-debug or orcharhino-debug.